.ci/build-docker-images.groovy

```diff
@@ -25,7 +25,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    cron 'H H(0-5) * * 1-5'
+    cron '@daily'
```

Comment:
do not use daily, it will run on weekends, and 24h after the last execution so the time is not fixed.

Reply:
Ok, I used what we added in the apm-pipeline-library for building images: https://github.com/elastic/apm-pipeline-library/blob/master/.ci/jobs/beats-docker-images-pipeline.yml#L27

What is the recommended cron trigger? Maybe weekly/monthly is enough

Reply:
'H H(0-5) * * 1-5' Monday to Friday anytime between 00:00 AM to 05:00 AM this is fine, the key is to set a range (H(0-5)) of hours where the build can run, Jenkins will manage to not overload the instance in that range.

Reply:
Ok, this is what we use for the nightly scheduled pipelines. Will use what we have in there